PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -117,7 +120,7 @@ public SparkSubmissionParameter getSubmissionParameter() {
     }
 
     public void setSubmissionParameters(SparkSubmissionParameter submissionParameters){
-        this.submissionParameter = submissionParameter;
+        this.submissionParameter = submissionParameters;
```

Comment:
This line is fix in merged pr https://github.com/Microsoft/azure-tools-for-java/pull/481.
It has conflicts now.